tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "lib": [
+      "DOM",
+      "ES5",
+      "ScriptHost",
```

Comment:
It's a standard include w/ an ES5 target. It's possible we don't need it.

https://github.com/Microsoft/TypeScript/blob/master/lib/lib.scripthost.d.ts